boards/tsi/skyp/tsi_skyp_m85.dts

```diff
@@ -89,15 +89,15 @@
 		zephyr,memory-region = "SRAM1";
 	};
 
-	soc {
+	/*soc {
 		peripheral@71000000 {
 			#address-cells = <1>;
 			#size-cells = <1>;
 			ranges = <0x0 0x71000000 0x1effffff>;
 
 			 #include "tsi_skyp_m85-common.dtsi"
```

Comment:
Should this dtsi file be included (moved out of this block)? Your comment block hides this currently.

Reply:
This inclusion of the dtsi file was only for this soc block. Not including it does not have major impact till we enable another IO.